frontend/src/components/comparison-popup/styles.module.scss

```diff
@@ -192,7 +192,3 @@ form > div:nth-child(2) {
     height: 150px;
   }
 }
-
-.noOptions {
-  padding-left: 5px;
-}
```

Comment:
Please turn this back, it was a necessary code.

![Screenshot_3](https://user-images.githubusercontent.com/49813216/191576221-26f1437f-f246-49ab-917f-b554989f18d1.png)